setup.py

```diff
@@ -1,14 +1,14 @@
 from setuptools import setup
 setup(
   name='operator-courier',
-  packages=['command'],
+  packages=['operatorcourier'],
```

Comment:
:heart: I wanted to propose name change, but you were faster. It sounds much better.